corehq/ex-submodules/casexml/apps/case/xform.py

```diff
@@ -3,6 +3,8 @@
 
 import itertools
 from casexml.apps.case.const import UNOWNED_EXTENSION_OWNER_ID
+# HACK: We need this import to initialize signal receivers. This should be handled in a different manner.
+import casexml.apps.case.signals  # noqa: F401
```

Comment:
I think the Django AppConfig `ready` method is the standard place to put this, e.g. https://github.com/dimagi/commcare-hq/blob/d6034c9043fdfae6595117a86bbbb7765de0a8fd/corehq/apps/hqadmin/app_config.py#L4-L8